messaging/content-templates/create-carousel-template/output/create-carousel-template.json

```diff
@@ -0,0 +1 @@
+{"account_sid": "ACXXXXXXXX", "date_created": "2024-07-24T20:35:59Z", "date_updated": "2024-07-24T20:35:59Z", "friendly_name": "twilio_deal", "language": "en", "links": {"approval_create": "https://content.twilio.com/v1/Content/HXxxxxxxxxx/ApprovalRequests/whatsapp", "approval_fetch": "https://content.twilio.com/v1/Content/HXxxxxxxxxx/ApprovalRequests"}, "sid": "HXxxxxxxx", "types": {"twilio/carousel": {"body": "New {{1}} merch just dropped! \ud83d\udc40", "cards": [{"actions": [{"id": "want_hoodie", "index": 0, "title": "I want it!", "type": "QUICK_REPLY"}, {"title": "I'm taking this!", "type": "URL", "url": "https://sienna-grasshopper-3262.twil.io/assets/hoodie.jpeg"}], "body": "Warm as owl feathers.", "media": "https://sienna-grasshopper-3262.twil.io/assets/hoodie.jpeg", "title": "Twilio Hoodie"}, {"actions": [{"id": "want_tote", "index": 0, "title": "I want it!", "type": "QUICK_REPLY"}, {"title": "Take the tote!", "type": "URL", "url": "https://sienna-grasshopper-3262.twil.io/assets/tote.jpeg"}], "body": "Carry a little more.", "media": "https://sienna-grasshopper-3262.twil.io/assets/tote.jpeg", "title": "Twilio Tote"}, {"actions": [{"id": "want_hat", "index": 0, "title": "I want it!", "type": "QUICK_REPLY"}, {"title": "Hand me the hat!", "type": "URL", "url": "https://sienna-grasshopper-3262.twil.io/assets/hat.jpeg"}], "body": "Stay in the shade.", "media": "https://sienna-grasshopper-3262.twil.io/assets/hat.jpeg", "title": "Twilio Bucket Hat"}, {"actions": [{"id": "want_mug", "index": 0, "title": "I want it!", "type": "QUICK_REPLY"}, {"title": "Make me a mug!", "type": "URL", "url": "https://sienna-grasshopper-3262.twil.io/assets/mug.jpeg"}], "body": "Sip a little.", "media": "https://sienna-grasshopper-3262.twil.io/assets/mug.jpeg", "title": "Twilio Mug"}]}}, "url": "https://content.twilio.com/v1/Content/HX2dcc3a9a5c8f4b428e3ada5a7e9dbee1", "variables": {"1": "Twilio"}}
```

Comment:
Do you want to prettify this, @benparj98 ? https://jsonformatter.org/json-pretty-print

Actually @CristhianMotoche , does this automatically get fixed when ingested by the platform?

Reply:
I just went ahead and made it pretty!